translations/pt.yml

```
@@ -0,0 +1,3280 @@
+  <strong>Etapa 2 de 3</strong>. Submissão enviada para avaliação, <strong>aguarde um retorno
+  </strong>
+instr-review__student-third-step: >-
+  "<strong>Etapa 3 de 3</strong>. Você recebeu a <a href=""{url}"">avaliação</a> do instrutor.
+  Resultado: <strong>{result}%</strong>"
+instr-review__teacher-notice: >-
+  {count, plural, =1 {<b>Avaliar # submissão</b>, os alunos receberão pontos} ​​other {<b> Avaliar #
+  submissões</b>, os alunos receberão pontos} =0 {Todas as avaliações foram enviadas.}}
+instr-review__teacher-start: Iniciar avaliação
+instruction__add-rubric-button: Adicionar um novo apontamento
+instruction__delete: Remover avaliação
+instruction__description-help:
+  Você pode adicionar aqui qualquer informação que possa ser útil para os avaliadores.
+instruction__frozen-description: >-
+  A avaliação já começou. Você pode alterar as descrições. Você não pode alterar a pontuação e
+  adicionar/remover apontamentos.
+instruction__header: Regras de avaliação
+instruction__max-point: Pontuação máxima
+instruction__max-point-placeholder: "entre {greater} e {less}"
+instruction__reviewcount-help: >-
+  Os alunos recebem a pontuação para a atividade somente ao receber um número necessário de
+  avaliações em suas submissões.
+instruction__reviews-required: "Número necessário de revisões para cada submissão:"
+instruction__rubric: Apontamento de pontuação
+instruction__rubric-deleted: (Excluído)
+instruction__rubric-help: >-
+  Os apontamentos de pontuação podem ajudar os alunos a se tornarem avaliadores atenciosos no
+  seu e no trabalho de outros.
+instruction__rubric-template: Classifique esta submissão de 0 a 3
+instruction__strategy-help: >-
+  Você pode avaliar as submissões dos alunos, para obterem uma pontuação mais objetiva.
+  Você também pode ativar a opção que permite que os alunos avaliem uns aos outros.
+instruction__strategy-instructor: Somente instrutores (professores)
+instruction__strategy-instructor-tooltip: Somente professores (você e os demais autores desta lição)
+instruction__strategy-label: Quem avaliará as submissões dos alunos?
+instruction__strategy-peer: Alunos e instrutores (professores)
+instruction__strategy-peer-tooltip: Alunos entre si. O professor (você) também pode avaliar
+instruction__teacher-comment: Comentário do professor
+instruction__will_be_deleted: >-
+  A avaliação está marcada para exclusão!<br> Ela será excluída depois que você pressionar
+  "Salvar". Clique aqui para desfazer.
+instruction_urgent:
+  btn: Concluir agora
+  btn_expl: A pontuação será dada com base nas avaliações recebidas no momento.
+  fail: "Falha ao concluir a avaliação, tente novamente mais tarde"
+  finished_expl: >-
+    {count, plural, =1 {Resultado da avaliação: {percentage}%. Por iniciativa do instrutor, a
+    pontuação foi baseada em uma avaliação única {date}.} other {Resultado da avaliação:
+    {percentage}%. Por iniciativa do instrutor, a pontuação foi baseada em # avaliações {date}.}
```

Fix of `Expected "=", "\\#", "\\\\", "\\u", "\\{", "\\}", "}" or [^{}\\\0-\x1F\x7f \t\n\r] but end of input found.`:
```suggestion
    {percentage}%. Por iniciativa do instrutor, a pontuação foi baseada em # avaliações {date}.}}
```